act-rules-format.bs

```diff
@@ -101,132 +105,150 @@ An ACT Rule MUST have a unique identifier that can be any unique text, such as p
 
 <div class=note>
-**Note**: In WCAG 2.1, success criteria do not evaluate to `passed`, `failed` or `inapplicable`. Rather they can be *satisfied* (or not). (See the [WCAG 2.1 definition: satisfies a success criterion](https://www.w3.org/TR/WCAG21/#dfn-satisfies).) If a success criterion is *not satisfied* a web page can only conform if there is a conforming alternative version, as described in [WCAG 2.1 Conformance Requirement 1](https://www.w3.org/TR/WCAG21/#cc1).
+  <p>**Note:** In WCAG 2.1, success criteria do not evaluate to `passed`, `failed` or `inapplicable`. Rather they can be *satisfied* (or not). (See the [WCAG 2.1 definition: satisfies a success criterion](https://www.w3.org/TR/WCAG21/#dfn-satisfies).) If a success criterion is *not satisfied* a web page can only conform if there is a conforming alternative version, as described in [WCAG 2.1 Conformance Requirement 1](https://www.w3.org/TR/WCAG21/#cc1).</p>
 </div>
 
 <div class=example>
-  <p><strong>Example:</strong> Accessibility Requirements Mapping for a rule that tests if an image button has an accessible name:
+  <p>**Example:** Accessibility Requirements Mapping for a rule that tests if an image button has an accessible name:
   <blockquote><ul>
-    <li><a href="https://www.w3.org/TR/WCAG21/#non-text-content">
-        Success Criterion 1.1.1: Non-text content
-      </a><ul>
-        <li><strong>Required for conformance</strong> to WCAG 2.0 and WCAG 2.1 level A</li>
+    <li>
+      [Success Criterion 1.1.1: Non-text content](https://www.w3.org/TR/WCAG21/#non-text-content)
+      <ul>
+        <li>**Required for conformance** to WCAG 2.0 and WCAG 2.1 level A</li>
         <li>Outcome mapping: <ul>
-          <li>Failed outcome: not satisfied</li>
-          <li>Passed outcome: further testing is needed</li>
+          <li>Any failed outcomes: not satisfied</li>
+          <li>All passed outcomes: further testing is needed</li>
           <li>Inapplicable outcome: further testing is needed</li>
         </ul></li>
       </ul>
     </li>
-    <li><a href="https://www.w3.org/TR/WCAG21/#name-role-value">
-        Success Criterion 4.1.2: Name, Role, Value
-      </a><ul>
-        <li><strong>Required for conformance</strong> to WCAG 2.0 and WCAG 2.1 level A</li>
+    <li>
+      [Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value)
+      <ul>
+        <li>**Required for conformance** to WCAG 2.0 and WCAG 2.1 level A</li>
         <li>Outcome mapping:<ul>
-          <li>Failed outcome: not satisfied</li>
-          <li>Passed outcome: further testing is needed</li>
+          <li>Any failed outcomes: not satisfied</li>
+          <li>All passed outcomes: further testing is needed</li>
```

Comment:
I was thinking about this as well, but there is an example in the next section that does map to satisfy. Not sure if we would be adding to many examples if we included another, so maybe reworking the example to show the different outcome mappings instead of just repeating the same pattern.